src/modals/SendModal/TransactionForm.tsx

```diff
@@ -131,10 +139,10 @@ const SendModalTransactionForm = ({ data, onSubmit, onCancel }: TransactionFormP
           </InfoBoxStyled>
         )}
       </ModalContent>
-      <ExpandableSectionStyled sectionTitleClosed="Advanced settings">
+      <ExpandableSectionStyled sectionTitleClosed="Gas">
         <Input
           id="gas-amount"
-          placeholder="Gas amount"
+          placeholder="Amount"
```

Comment:
I'd leave it as it was, same below. Better be explicit, it doesn't cost us much real estate imo

Reply:
I found it was redundant, having "Gas" twice in close proximity. @mvaivre what do you think of cases like this? Leave the redundancy?

Reply:
Let's see how it actually looks like.